scopus/scopus_author.py

```diff
@@ -131,7 +131,8 @@ def __init__(self, author_id, refresh=False, refresh_aff=False, level=1):
         Parameters
         ----------
         author_id : str or int
-            The ID of the author to search for.
+            The ID of the author to search for. Optionally expressed
+            as an Elsevier eid (i.e., in the form 9-s2.0-nnnnnnnn).
```

Comment:
Very good! I never encountered this format.  Could you please write EID instead of eid?

Reply:
Of course (by the way, EID is the object id in the Elsevier domain: it refers to any kind of object, e.g. 9-s2.0 are authors whereas 2-s2.0 are papers).